tests/test_holidays.py

```diff
@@ -0,0 +1,23 @@
+from app.database.models import Event
```

Comment:
An idea for another test-case: get file with wrong format.

Reply:
Since im using regex - wrong format will just yield zero results. ill add another test with wrong structre.